### DISCUSSION TITLE
RBAC for TwentyCRM

### DISCUSSION THREAD

**Discussion by beginAI** [twentyhq/twenty#8998] [Engineering vision]
We are planning to use TwentyCRM for our sales usecase, however the unavailability of RBAC makes it a difficult choice. Specially since we will be onboarding external teams for sales. Are there any known plugins/tools that can be integrated with TwentyCRM to support RBAC? Any visibility on the timelines when this feature will be developed by the TwentyCRM team?

**Comment by MrSuddenJoy** [comment]
@beginAI also waiting for this to be implemented.

**Comment by BOHEUS** [comment]
@beginAI @MrSuddenJoy RBAC is available behind Enterprise license

**Reply by MrSuddenJoy** [reply]
@BOHEUS This is perfect example why Ive ditched Twenty, year+ wait time for response.

**Reply by BOHEUS** [reply]
@MrSuddenJoy sorry about that, discussions weren't checked often and properly, that's why there were so many of them that are outdated and now I cleared them up 

But to be fair, there's a road map available for a year now (which was announced if I remember correctly and is in pinned issue)
